lib/chef-dk/policyfile_lock.rb

```diff
@@ -141,6 +146,7 @@ def to_lock
         lock["default_attributes"] = default_attributes
         lock["override_attributes"] = override_attributes
         lock["solution_dependencies"] = solution_dependencies.to_lock
+        lock["included_policy_locks"] = included_policy_locks
```

Comment:
We should move this up, so it appears higher in the generated lockfile. Lockfiles are intended to be inspected by users, but the `solution_dependencies` part is mostly for internal use, so the "good stuff" should go up top.